src/games/ecards.py

```diff
@@ -131,76 +133,33 @@ async def execute_round(self, leader: str, prompt: str, hands: dict):
         :param prompt: the prompt for the round
         :param hands: the players' hands
         """
-        # announce new round
-        em_text = f"Starting new round -- {self.players[leader]} is leader.\nThis round's prompt: \n**{prompt}**"
-        message = await self.channel.send(embed=self.embed(em_text))
-
-        # immutable pids
-        pids = list(self.players.keys())
-
-        # construct content
-        content_dict = {
-            pid: (
-                self.embed(
-                    f"This round's prompt: {prompt}\nYou're the leader for this round - sit back and relax!"
-                )
-                if pid == leader
-                else self.embed(f"**{prompt}**")
-            )
-            for pid in pids
-        }
-
         # get all message responses
-        tasks = []
-        for pid in pids:
-            # get channel
-            dm_channel = await self.players[pid].create_dm()
-
-            if pid != leader:
-                # make pipeline
-                ipl = InteractionPipeline(ChoiceInteraction(*hands[pid], max_votes=1))
-
-                # send message by storing coroutine
-                tasks.append(
-                    ipl.send_and_watch(dm_channel, content_dict[pid], timeout=31)
-                )
-            else:
-                # leader
-                tasks.append(dm_channel.send(embed=content_dict[pid]))
-
-        # await routines
-        dm_response = {
-            # ensure type consistent output
-            pid: resp if pid != leader else {}
-            for pid, resp in zip(pids, await asyncio.gather(*tasks))
-        }
+        root_embed = self.embed(f"Starting new round -- {self.players[leader]} is leader.\nThis round's prompt: \n**{prompt}**")
+        view = CardsGetPromptView(
+            root_embed,
+            self.game_name,
+            leader,
+            hands,
+            delete_after=True,
+            timeout=31,
```

Comment:
Do we not have a config variable for these timeouts?

Reply:
No, only for the player gather timeout. I can add one if you want